docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -685,77 +583,56 @@ secret/application,dev
+The #_property_overrides[property overrides] feature can also be used for setting global defaults, with placeholders applications
 allowed to override them locally.
 
-TIP: With the "native" profile (local file system backend) it is
-recommended that you use an explicit search location that isn't part
-of the server's own configuration. Otherwise the `application*`
-resources in the default search locations are removed because they are
-part of the server.
+TIP: With the "`native`" profile (a local file system backend) , you should use an explicit search location that is not part of the server's own configuration.
+Otherwise, the `application*` resources in the default search locations get removed because they are part of the server.
 
+[[spring-cloud-config-server-vault-server]]
 ===== Vault Server
 
-When using Vault as a backend you can share configuration with
-all applications by placing configuration in
-`secret/application`.  For example, if you run this Vault command
+When using Vault as a backend, you can share configuration with all applications by placing configuration in `secret/application`.
+For example, if you run the following Vault command, all applications using the config server will have the properties `foo` and `baz` available to them:
 
 [source,sh]
 ----
 $ vault write secret/application foo=bar baz=bam
 ----
 
-All applications using the config server will have the properties
-`foo` and `baz` available to them.
-
 ==== JDBC Backend
 
-Spring Cloud Config Server supports JDBC (relation database) as a
-backend for configuration properties. You can enable this feature by
-adding `spring-jdbc` to the classpath, and using the "jdbc" profile,
-or by adding a bean of type `JdbcEnvironmentRepository`. Spring Boot
-will configure a data source if you include the right dependencies on
-the classpath (see the user guide for more details on that).
-
-The database needs to have a table called "PROPERTIES" with columns
-"APPLICATION", "PROFILE", "LABEL" (with the usual `Environment`
-meaning), plus "KEY" and "VALUE" for the key and value pairs in
-`Properties` style. All fields are of type String in Java, so you can
-make them `VARCHAR` of whatever length you need. Property values
-behave in the same way as they would if they came from Spring Boot
-properties files named `{application}-{profile}.properties`, including
-all the encryption and decryption, which will be applied as
-post-processing steps (i.e. not in the repository implementation
-directly).
+Spring Cloud Config Server supports JDBC (relational database) as a backend for configuration properties.
+You can enable this feature by adding `spring-jdbc` to the classpath and using the `jdbc` profile or by adding a bean of type `JdbcEnvironmentRepository`.
+If you include the right dependencies on the classpath (see the user guide for more details on that), Spring Boot configures a data source.
+// TODO Which user guide? When we know that, we should add a link to it.
```

Comment:
Maybe this, https://spring.io/guides/gs/relational-data-access/?  @dsyer originally wrote this